ui-admin/src/study/StudyContent.tsx

```diff
@@ -85,45 +79,35 @@ function StudyContent({ studyEnvContext }: {studyEnvContext: StudyEnvContextT})
   const consentSurveyStableIds = getUniqueStableIdsForType(configuredSurveys, 'CONSENT')
   const adminFormStableIds = getUniqueStableIdsForType(configuredSurveys, 'ADMIN')
   const documnetRequestStableIds = getUniqueStableIdsForType(configuredSurveys, 'DOCUMENT_REQUEST')
+  const preEnrollStableIds = getUniqueStableIdsForType(configuredSurveys, 'PRE_ENROLL')
 
   return <div className="container-fluid px-4 py-2">
     { renderPageHeader('Forms & Surveys') }
     <LoadingSpinner isLoading={isLoading}>
       <div className="col-12">
         { currentEnv.studyEnvironmentConfig.initialized && <ul className="list-unstyled">
           <li className="mb-3 rounded-2 p-3" style={{ background: '#efefef' }}>
-            <h6>Pre-enrollment Questionnaire</h6>
+            <h2 className="h6">Pre-Enroll Surveys</h2>
             <div className="flex-grow-1 pt-3">
-              {preEnrollSurvey && <ul className="list-unstyled">
-                <li className="d-flex align-items-center">
-                  <Link to={`preEnroll/${preEnrollSurvey.stableId}?readOnly=${isReadOnlyEnv}`}>
-                    {preEnrollSurvey.name} <span className="detail">v{preEnrollSurvey.version}</span>
-                  </Link>
-
-                  {!isReadOnlyEnv && <div className="nav-item dropdown ms-1">
-                    <IconButton icon={faEllipsisH} data-bs-toggle="dropdown"
-                      aria-expanded="false" aria-label="configure pre-enroll menu"/>
-                    <div className="dropdown-menu">
-                      <ul className="list-unstyled">
-                        <li>
-                          <button className="dropdown-item"
-                            onClick={() => alert('To remove a pre-enroll survey, contact support')}>
-                              Remove
-                          </button>
-                        </li>
-                      </ul>
-                    </div>
-                  </div>}
-                </li>
-              </ul>}
-              {(!preEnrollSurvey && !isReadOnlyEnv) && <Button variant="secondary"
-                data-testid={'addPreEnroll'}
-                onClick={() => {
-                  setShowCreatePreEnrollModal(!showCreatePreEnrollSurveyModal)
+              <SurveyEnvironmentTable
```

Comment:
For pre-enrolls, should we hide certain task-related functionality like "see participant assignment", "assign to {studyEnv} participants", and other settings like recurrence interval?

Reply:
yeah, right on.  We can tackle that in the subsequent schema and admin cleanup PR